src/components/EditCoursePage/CollapsibleCourseRun.test.jsx

```diff
@@ -109,6 +111,32 @@ describe('Collapsible Course Run', () => {
     expect(shallowToJson(component)).toMatchSnapshot();
   });
 
+  it('renders correctly variant_id field for external course\'s course run', () => {
+    const courseInfo = {
+      data: {
+        product_source: {
+          slug: 'test-source',
+          name: 'Test Source',
+          description: 'Test Source Description',
+        },
+        course_type: EXECUTIVE_EDUCATION_SLUG,
+      },
+    };
+    const component = shallow(<CollapsibleCourseRun
+      languageOptions={languageOptions}
+      pacingTypeOptions={pacingTypeOptions}
+      courseRun={unpublishedCourseRun}
+      courseId="test-course"
+      courseUuid="11111111-1111-1111-1111-111111111111"
+      type="8a8f30e1-23ce-4ed3-a361-1325c656b67b"
+      currentFormValues={currentFormValues}
+      courseRunTypeOptions={courseRunTypeOptions}
+      index={1}
+      courseInfo={courseInfo}
+    />);
+    expect(shallowToJson(component)).toMatchSnapshot();
```

Comment:
Do we need to use `shallowToJson` in this example? Why can't we simply do that without `shallowToJson`?

Reply:
We can do it without using `shallowToJson`. I included it here because it is utilized in the other tests of the same file. 
However, I've updated the test to verify this via both (snapshot, find) methods